notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,7 +15,8 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # save output in netcdf format
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
fixed.